pkg/controller/releasemanager/scaling/linear.go

```diff
@@ -7,35 +7,36 @@ import (
 // LinearScale scales a ScalableTarget linearly
 func LinearScale(st ScalableTarget, max uint32, t time.Time) uint32 {
 	current := st.CurrentPercent()
+
 	if max <= 0 {
 		return 0
 	}
-	// Only scale if previous target is met.
-	if !st.IsReconciled() {
-		if current > max {
-			return max
-		}
-		return current
-	}
+
 	if st.Max() < max {
 		max = st.Max()
 	}
+
+	// Scaling down
+	if current > max {
+		return max
+	}
+
+	// Wait longer to scale
 	deadline := st.LastUpdated().Add(st.Delay())
 	if deadline.After(t) {
-		if current > max {
-			return max
-		}
 		return current
 	}
 
 	increment := st.Increment()
-	if st.PeakPercent() >= 100 {
```

Comment:
got rid of this, is strange... we can revisit later. It was meant for quick rollbacks but it's not well thought out.